pset-functions.js

```diff
@@ -75,6 +96,41 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+let score = 0
+let i = 0
+function scrabbleScore(word) {
+  for (i = 0; i < word.length; i++) {
+    if (word.charAt(i) === 'a' || word.charAt(i) === 'e' || word.charAt(i) === 'i' || word.charAt(i) === 'o' || word.charAt(i) === 'u' || word.charAt(i) === 'l' || word.charAt(i) === 'n' || word.charAt(i) === 'r' || word.charAt(i) === 's' || word.charAt(i) === 't') {
```

Comment:
Another really great use of it/else blocks to catch all possible scenarios and great use of a for loop. Fun tip: you can achieve the same thing with "word.charAt(i)" as you can with "word[i]" (strings have an index that corresponds to each character/space/symbol/etc.). Be a little careful with declaring variables outside of the function you intend to use them in (it creates a global variable as a result). For your benefit as well, I like to attach alternate solutions so you have many tools in your arsenal; please find a solution using .includes on an array along with the forEach method [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L97-L98).